modules/installation-requirements-user-infra.adoc

```diff
@@ -80,6 +102,65 @@ Each cluster machine must meet the following minimum requirements:
 
 |===
 
+ifeval::["{context}" == "installing-ibm-z"]
+[id="minimum-ibm-z-system-requirements_{context}"]
+== Minimum IBM Z system requirements
+
+=== Hardware requirements
```

Comment:
All headings need explicit anchors or to be tagged as discrete.

Reply:
Fixed in b5ec34469f8d2e53be5c28547b30fd5d4339d20b